packages/account-v2/src/modules/AddressFields/validations.ts

```diff
@@ -1,13 +1,13 @@
 import * as Yup from 'yup';
 
 const regexChecks = {
-    address_details: {
-        address_city: /^\p{L}[\p{L}\s'.-]{0,99}$/u,
-        address_line_1: /^[\p{L}\p{Nd}\s'.,:;()\u00b0@#/-]{1,70}$/u,
-        address_line_2: /^[\p{L}\p{Nd}\s'.,:;()\u00b0@#/-]{0,70}$/u,
-        address_postcode: /^[a-zA-Z0-9\s-]{0,20}$/,
-        address_state: /^[\w\s'.;,-]{0,99}$/,
-        non_jersey_postcode: /^(?!\s*je.*)[a-z0-9\s-]*/i,
+    addressDetails: {
```

Comment:
Suggestion: I see the file is already in `AddressFields` folder. In this case adding `addressDetails` in the `regexChecks` is kinda unnecessary.